src/Projection/Projectionist/DefaultProjectionist.php

```diff
@@ -593,4 +590,92 @@ private function handleRetryProjections(ProjectionCollection $projections): void
             );
         }
     }
+
+    private function handleFromNowProjections(ProjectionCollection $projections): void
+    {
+        $latestIndex = null;
+
+        foreach ($projections->filterByProjectionStatus(ProjectionStatus::Booting) as $projection) {
+            $projector = $this->projector($projection->id());
+
+            if (!$projector) {
+                continue;
+            }
+
+            $metadata = $this->metadataFactory->metadata($projector::class);
+
+            if (!$metadata->fromNow) {
+                continue;
+            }
```

Comment:
maybe extract this filtering in a private method to increase readability and reduce the complexity in this method?

Reply:
I will do this in a following PR as I have started building the Projectionist differently.